packages/cli/test/unit/commands/build/index.test.ts

```diff
@@ -1025,6 +1025,36 @@ describe('build', () => {
     }
   });
 
+  it('should set VERCEL_PROJECT_SETTINGS_ environment variables', async () => {
+    if (process.platform === 'win32') {
+      // this test runs a build command with `mkdir -p` which is unsupported on Windows
+      console.log('Skipping test on Windows');
+      return;
+    }
+
+    const cwd = fixture('project-settings-env-vars');
+    const output = join(cwd, '.vercel/output');
+    try {
+      process.chdir(cwd);
+      const exitCode = await build(client);
+      expect(exitCode).toEqual(0);
+
+      const contents = await fs.readFile(
```

Comment:
`fs.readJson()`?

Reply:
Done in 6b968c20bafe37cadfb7c9af0800cde53a1e8630